app/azure/Dashboard/DashboardView.tsx

```diff
@@ -0,0 +1,71 @@
+import { useEnvironmentId } from '@/portainer/hooks/useEnvironmentId';
+import { PageHeader } from '@/portainer/components/PageHeader';
+import { error as notifyError } from '@/portainer/services/notifications';
+import { r2a } from '@/react-tools/react2angular';
+
+import { aggregateResourceGroups } from '../utils';
+import { useResourceGroups, useSubscriptions } from '../queries';
+
+import { DashboardItem } from './DashboardItem';
+
+export function DashboardView() {
+  const environmentId = useEnvironmentId();
+
+  const {
+    data: subscriptions,
+    isLoading: isLoadingSubscriptions,
+    error: subscriptionsError,
+    isError: isErrorSubscriptions,
+  } = useSubscriptions(environmentId);
```

Comment:
can be easier to name it `subscriptionsQuery`
```suggestion
 const subscriptionQuery  = useSubscriptions(environmentId);
```

then you can do `subscriptionQuery.isLoading` (`/isError/data`)

Reply:
thanks for this suggestion, it is a lot cleaner